org.nongnu.Xlog.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: org.nongnu.Xlog
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: xlog
+rename-desktop-file: xlog.desktop
+finish-args:
+  - --device=all
```

Comment:
I forgot to mention that access to all devices is needed for Xlog to be able to communicate with radio devices using hamlib.